crates/sb_workers/event_worker.js

```diff
@@ -3,35 +3,33 @@ const { SymbolAsyncIterator } = primordials;
 const core = globalThis.Deno.core;
 
 class SupabaseEventListener {
-    async nextEvent() {
-        try {
-            return await core.opAsync("op_event_accept");
-        } catch (e) {
-            console.error(e);
-        }
-    }
-
-    #buildEvent(event) {
-        const rawEvent = event['Event'];
-        const eventType = Object.keys(rawEvent)[0];
-        return {
-            deployment_id: "",
-            timestamp: new Date().toISOString(),
-            event_type: eventType,
-            event: rawEvent[eventType],
-            execution_id: "",
-            region: "",
-            context: undefined
-        }
-    }
-
     [SymbolAsyncIterator]() {
-        const scopedClass = this;
         return {
             async next() {
```

Comment:
@andreespirela I simplified the symbol iterator a bit. Was there any specific reason we had the `nextEvent` function?